style.css

```diff
@@ -438,3 +468,283 @@ a {
   margin-bottom: 30px;
   text-align: center;
 }
+
+/* The desktop design */
+@media (min-width: 650px) and  (max-width: 1100px) {
```

Comment:
Please use the media query of min-width of any point between `992px` to `1100`
for example:
```
@media screen and (min-width: 990px) {
/*
.
.
*/
}
```
instead of the media query you use 
```
@media (min-width: 650px) and  (max-width: 1100px) {
}
```